site/SiteFooter.tsx

```diff
@@ -89,10 +89,10 @@ export const SiteFooter = (props: SiteFooterProps) => (
                             </li>
                             <li>
                                 <a
-                                    href="/about/how-to-use-our-world-in-data"
+                                    href="/faqs"
                                     data-track-note="footer_navigation"
                                 >
-                                    How to use
+                                    FAQs
```

Comment:
I think it's nicer to write this out as `Frequently asked questions`

Reply:
The link is `FAQs` on the about page, so I opted for consistency and also there isn't much space in the footer column, so the longer link wraps on a medium screen. Do you feel strongly that we shouldn't use the acronym?

Reply:
You're right, makes sense. Good to go ahead then :)